services/src/main/java/org/fao/geonet/api/records/MetadataWorkflowApi.java

```diff
@@ -1310,4 +1316,23 @@ private void changeMetadataStatus(ServiceContext context, AbstractMetadata metad
         listOfStatusChange.add(metadataStatusValue);
         sa.onStatusChange(listOfStatusChange);
     }
+
+
+    /**
+     * Checks if the user profile is allowed to view metadata history status.
+     *
+     * @param userSession
+     */
+    private void checkUserProfileToViewMetadataHistory(UserSession userSession) {
+        if (userSession.getProfile() != Profile.Administrator) {
+            String allowedUserProfileToImportMetadata =
+                org.apache.commons.lang.StringUtils.defaultIfBlank(settingManager.getValue(Settings.METADATA_HISTORY_ACCESS_LEVEL), Profile.Editor.toString());
+
+            // Is the user profile is higher than the profile allowed?
```

Comment:
I think you should change **is higher** to **higher**
`// Is the user profile higher than the profile allowed?`

Reply:
Yes, I fixed it.